shell/platform/android/io/flutter/plugin/editing/ListenableEditingState.java

```diff
@@ -171,6 +207,48 @@ public void removeEditingStateListener(EditingStateWatcher listener) {
     }
   }
 
+  @Override
+  public SpannableStringBuilder append(char text) {
+    Log.e("DELTAS", "insert #1 is called");
+    return append(String.valueOf(text));
```

Comment:
I guess this is the actual implementation of this `append`, you're just overriding it in order to be able to log here?  And below.

Reply:
Oh and you don't seem to need to override these to determine the type, you can just infer it inside of `replace`.

Reply:
Yeah all these overrides where just to see if we actually hit these calls.